Makefile

```diff
@@ -12,18 +12,28 @@ else
 BUILD_RUN = npm run build
 endif
 
+WEBLATE_REPO=cockpit-composer-weblate
+WEBLATE_REPO_URL=git@github.com:cockpit-project/cockpit-composer-weblate.git
+WEBLATE_REPO_BRANCH=master
+
 all: npm-install
 	NODE_ENV=$(NODE_ENV) $(BUILD_RUN)
 
-npm-install-zanata: npm-install
-	npm install --no-save zanata-js
+$(WEBLATE_REPO):
+	git clone --depth=1 $(WEBLATE_REPO_URL) $(WEBLATE_REPO)
+	git -C $(WEBLATE_REPO) checkout $(WEBLATE_BRANCH)
```

Comment:
Nice! I love this method. Thanks @marusak! 
One comment here. `$(WEBLATE_BRANCH)` should be `$(WEBLATE_REPO_BRANCH)`

Reply:
Indeed, thanks for spotting! Fixed